pkg/network/nr/net_resource.go

```diff
@@ -308,17 +316,24 @@ func (nr *NetResource) createNetNS() error {
 	if err != nil {
 		return err
 	}
-	netNS.Close()
+	defer netNS.Close()
+	err = netNS.Do(func(_ ns.NetNS) error {
+		if _, err := sysctl.Sysctl("net.ipv6.conf.all.forwarding", "1"); err != nil {
+			return err
+		}
+		if err := ifaceutil.SetLoUp(); err != nil {
+			return err
+		}
+		return nil
+	})
 	return nil
 }
 
-// createVethPair creates a veth pair inside the namespace of the
-// network resource and sends one side back to the host namespace
-// the host side of the veth pair is attach to the bridge of the network resource
-func (nr *NetResource) createVethPair() error {
-
+// We're no setting up a veth pair any more, instead, we use a macvlan
```

Comment:
Comment is not correct. 
Golang comment always starts with the name of the function e.g: `attachToNRBridge attaches a ..`.
Also don't describe it with thing like "any more". The code doesn't show any history so a reader have no clue what was here before. Instead describe why it does what it does.